src/Sarif/Schemata/sarif-2.1.0-rtm.5.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
rtm.6 files are copied from rtm.5.
rtm.5 files are reverted to pre-uuid change version.